sqlalchemy_continuum/manager.py

```diff
@@ -153,6 +158,36 @@ def reset(self):
 
         self.metadata = None
 
+    def after_begin(self, session, tx, conn):
+        if not self.options['versioning'] or not self.options['native_versioning']:
```

Comment:
I was thinking about refactoring these checks, now that we seem to have two distinct code-paths (native vs. not are almost entirely orthogonal).  This first pass was made to match the logic around it, for now, and I didn't want to move too much around to avoid any risk of breaking non-native versioning, but yea, it feels as if the checks should be in that dictionary of lifecycle functions above

Reply:
Do we want the option of turning either on/off? If we're doing `native_versioning`, should we have the triggers installed in our local test dbs as well?